hmx2/constants.py

```diff
@@ -1,20 +1,24 @@
 # ------ Pyth Network ------
```

Comment:
Let's think about breaking this file down as it will grows when new chain is added. Can do in the next pr if its gonna make this PR even more dirty.

constants (folder)
  --> contracts.py
  --> markets.py
  --> tokens.py
  --> common.py (for the rest that can't be group)